seqr/views/utils/variant_utils.py

```diff
@@ -83,3 +89,108 @@ def saved_variant_genes(variants):
         if gene:
             gene['locusListGuids'] = []
     return genes
+
+
+def _add_locus_lists(projects, genes, add_list_detail=False, user=None, is_analyst=None):
```

Comment:
copy-pasted from saved_variant_api.py, no changes other than making it private to this file